src/lesson3/hw_3_1.java

```diff
@@ -0,0 +1,43 @@
+package lesson3;
+
+import java.util.Scanner;
+
+    public class hw_3_1 {
```

Comment:
і тут зайвий таб, відповідно весь клас поплиив 
```suggestion
import java.util.Scanner;

public class hw_3_1 {
```